stores/tasks.ts

```diff
@@ -51,10 +52,24 @@ export const useTasksStore = defineStore('tasks', () => {
       )
     },
     onMutate({ positionUpdates }) {
-      positionUpdates.forEach(({ task, labels, rpb }) => {
-        task.labels = labels
-        task.rpb = rpb
-      })
+      if (!tasks.value) {
+        return
+      }
+
+      for (const { task, labels, rpb } of positionUpdates) {
+        const taskIndex = tasks.value.indexOf(task)
+        if (taskIndex === -1) {
+          return
+        }
+
+        const newTask = {
+          ...task,
+          labels,
+          rpb,
+        }
```

Comment:
We can make these one line and make it much smaller

Reply:
Done. Found a couple more case of this and reduced them also.